### ISSUE TITLE
Can't be used with Faraday 2

### ISSUE THREAD

**Issue by GarrisonD** [centrifugal/rubycent#22]
`cent` requires `faraday_middleware` that can be used only with `faraday 1.x`.

```
Could not find compatible versions

Because cent >= 2.0.0 depends on faraday_middleware >= 1.0, < 2.A
  and faraday_middleware >= 1.0.0.rc1 depends on faraday ~> 1.0,
  cent >= 2.0.0 requires faraday ~> 1.0.
So, because Gemfile depends on cent ~> 2.2
  and Gemfile depends on faraday ~> 2.9,
  version solving has failed.
```

**Comment by FZambia** [comment]
@GarrisonD hello,

Could you check https://github.com/centrifugal/rubycent/pull/23 ? Hopefully will fix the issue, I am not Ruby dev - so if you can review PR would be nice.

@prikha FYI

**Comment by GarrisonD** [comment]
Done ✅

**Comment by prikha** [comment]
Glad they dropped it, it was one of the most confusing parts of Faraday. ✅

**Comment by FZambia** [comment]
Thanks guys, released [v3.0.0](https://github.com/centrifugal/rubycent/releases/tag/v3.0.0).
